### DISCUSSION TITLE
BlackArch ARM mirrors Error 404

### DISCUSSION THREAD

**Discussion by UnknownDemonHacker** [BlackArch/blackarch#4101] [Support] [answered]
- [ ] I have searched open and closed issues for duplicates
- [ ] I am submitting a bug report for existing functionality that does not work as intended

----------------------------------------

### Bug description

when updating the system or syncing the mirrors error is shown even after changing the mirrors it shows the same error after further research this error is found only on arm devices like raspberry pi  

### Steps to reproduce

- pacman -Syy
          or
- pacman -S

**Actual result:**
 blackarch.db failed to download
error: failed retrieving file 'blackarch.db' from ftp.halifax.rwth-aachen.de : The requested URL returned error: 404
error: failed to synchronize all databases (failed to retrieve some files)

**Expected result:** 
the system should update or the mirrors are synced without any error
### Screenshots

![Screenshot from 2024-03-15 16-41-37](https://github.com/BlackArch/blackarch/assets/83639690/e2fc2b70-41a4-4b4d-b4c6-a39fc41daac3)

### Info for developers

**Device:** Raspberry Pi 3B+
**GNU/Linux distribution:** BlackArch on top of Arch Linux 
**Tool version:** 0.0.0

### Link to debug log

<! 
![Screenshot from 2024-03-15 16-41-37](https://github.com/BlackArch/blackarch/assets/83639690/c4d46028-6f73-4835-bc66-5eebe619efff)

**Comment by noraj** [comment, marked as answer]
> BlackArch on top of Arch Linux

ArchLinux ARM is not Arch Linux but the way. ArchLinux ARM is unofficial. ArchLinux works only on x86_64.

> after changing the mirrors

You changed BA mirrors or alarm mirrors?

Also `pacman -Syy` leads to [Partial upgrade](https://wiki.archlinux.org/title/System_maintenance#Partial_upgrades_are_unsupported).

An before syncing repos you have a [error: GPGME error: No data](https://wiki.archlinux.org/title/Pacman#error:_GPGME_error:_No_data).

**Reply by UnknownDemonHacker** [reply]
I am using alarm mirror, I stopped using BA mirror for now.
Sorry for the very late reply

**Comment by ikstream** [comment]
Hi,
Are you using the aarch64 image or the 32 bit image on the rpi 3B+?
Only the 64 bit is supported by blackarch.
Please provide the output of `uname -a`

**Reply by UnknownDemonHacker** [reply]
This is the output for `uname -a` 
`Linux Nyx 6.6.26-1-rpi #1 SMP Thu Apr 11 13:14:04 MDT 2024 armv7l GNU/Linux`

Sorry for the very late reply

**Reply by ikstream** [reply]
Looks like you are using the armv71 architecture, which is not supported by BlackArch at the moment.
